packages/devtools_app/lib/src/standalone_ui/vs_code/flutter_panel.dart

```diff
@@ -190,39 +196,54 @@ class _EditorConnectedPanelState extends State<_EditorConnectedPanel>
     );
   }
 
+  @override
+  void dispose() {
+    scrollController.dispose();
```

Comment:
This class has `AutoDisposeMixin`, does it help here?

Reply:
No, because we are not adding any listeners to this scroll controller. If we were, then those listeners would get automatically disposed if they were added with `addAutoDisposeListener`, but we'd still have to dispose the `scrollController` object manually. This dispose call is what would trigger any listeners on `scrollController` to get disposed by the AutoDispose logic.

Reply:
Ah, I thought we could just give it anything disposable (eg. the `ScrollController`) and have it dispose. Thanks for the clarification :)